cdisc_rules_engine/dataset_builders/base_dataset_builder.py

```diff
@@ -71,6 +72,17 @@ def get_dataset(self, **kwargs):
                 dataset_names=self.get_corresponding_datasets_names(),
                 **kwargs,
             )
+        elif (
```

Comment:
Could you please mention the reason of specifically filtering on CG0019. Shouldn't we be using a general approach so in future even if ids change or some other rule of same nature comes it still works?

Reply:
I was told we do not want to merge all supp datasets in the way this merge is performed.  I agree filtering on just the rule ID is narrow and not clean, reusable code but I am uncertain of what we do want to fitler into this merge logic on.  I believe @gerrycampion would be better able to advise on what we would want to filter on to control flow into this merge logic but he has been away.  I assumed we would revisit this filter logic later when he had returned.

Reply:
yes, we can revisit it later then